samplers/parser.go

```diff
@@ -69,12 +70,48 @@ func ConvertMetrics(m *protocol.Message) ([]UDPMetric, error) {
 		}
 		metrics = append(metrics, metric)
 	}
+	indicators, err := metricsFromIndicatorSpan(m)
+	if err == nil {
+		metrics = append(metrics, indicators...)
+	}
 	if len(invalid) != 0 {
 		return metrics, &invalidMetrics{invalid}
 	}
 	return metrics, nil
 }
 
+const timerMetric = "ssf.indicator_duration_ms"
```

Comment:
Prefixing with SSF feels a bit icky to me. What do you think of making it configurable? Something like `orgname.canonical_operation_duration_ms` seems pretty nice!

Alternatively, I kinda wanted to suggest a `printf` style option to interpolate the service name in, but am unsure how I feel about that.